src/index.js

```diff
@@ -0,0 +1,40 @@
+/* eslint global-require: 0 */
+import 'babel-polyfill';
+import React from 'react';
+import { render } from 'react-dom';
+import { browserHistory } from 'react-router';
```

Comment:
Do you think it'd be a good idea to change this line to `` import { browserHistory as history } from 'react-router'; ``?